README.md

```diff
@@ -295,6 +295,14 @@ The `data.inventory` document has the following format:
   * For namespace-scoped objects: `data.inventory.namespace[<namespace>][groupVersion][<kind>][<name>]`
      * Example referencing the Gatekeeper pod: `data.inventory.namespace["gatekeeper"]["v1"]["Pod"]["gatekeeper-controller-manager-d4c98b788-j7d92"]`
 
+### Customize Startup Behavior
+
+#### Allow retries when adding objects to OPA
+
+While objects are injested into the OPA cache, Gatekeeper prevents the validating webhook from serving requests.  This prevents the webhook from validating based on an incomplete set of policies.
```

Comment:
nit: this should make clear that this is only during bootstrapping.

Gatekeeper's webhook servers undergo a bootstrapping period during which they are unavailable until the initial set of resources (constraints, templates, synced objects, etc...) have been ingested. This prevents Gatekeeper's webhook from validating based on an incomplete set of policies. This wait-for-bootstrapping behavior can be configured.

Reply:
Done